### PR TITLE
A second incomplete version of the docs

### PR DESCRIPTION
**Pull request by RaulRPrado**
@orelgueta, this time I want to test the integration with the github pages. Can you have a look and let me know if the merging is fine? Again, only minimal changes to the code and a lot of changes to the docs directory.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -13,24 +8,26 @@ gammasim-tools
 What is gammasim-tools?
 =======================
 
-Prototype implementation of tools for the Simulation System of the `CTA Observatory <www.cta-observatory.org>`_
+gammasim-tools is a prototype of a software package that provides tools and
```
Comment (orelgueta) [inline docs/source/index.rst:11]: Can we make this claim at the moment in a public space? @GernotMaier?

**Review by orelgueta — APPROVED** [review]
Looks good. I didn't read everything super carefully since I am in a meeting (do you want me to?), but I doubt there are any real issues.

**Comment by RaulRPrado** [comment]
> Looks good. I didn't read everything super carefully since I am in a meeting (do you want me to?), but I doubt there are any real issues.

I don't think you need to read it carefully.

File: docs/source/conventions.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.. _Telescope Names:
+
+Telescope Names
+---------------
+
+The telescope names as used by gammasim-tools follow the pattern "Site-Class-Type", where:
+
+* "Site" is either "North" or "South";
+* "Class" is either "LST", "MST", "SCT" or "SST";
+* "Type" is a single number ONLY in case of a real telescope existing at the site or a string containing a "D" in case of any other telescope design.
```
[the remark this answers is not in the document]
Comment (RaulRPrado) [reply docs/source/conventions.rst:15]: It doesn't have anything. It's only recommended that contains a D.

**Comment by RaulRPrado** [comment]
I think you guys should better review the content of the documentation later, when it's complete.
This PR is an intermediate step in order to setup the integration with the gh-pages.

File: docs/source/conventions.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (orelgueta) [reply docs/source/conventions.rst:15]: From the text it reads as if it has to start with a "D". I would change it to say that it has to start with any string and then give the example we have with "D".
This is far from critical, just a clarity issue.

**Comment by orelgueta** [comment]
> I think you guys should better review the content of the documentation later, when it's complete.
> This PR is an intermediate step in order to setup the integration with the gh-pages.

Yeah, makes sense.

File: docs/source/conventions.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (RaulRPrado) [reply docs/source/conventions.rst:15]: Ok, I'll rewrite it later then. Thanks.

**Comment by RaulRPrado** [comment]
I'll merge it then.
The integration is working, so now I just need to finish the writing.

**Pull request merged**